packages/tizen_bundle/lib/tizen_bundle.dart

```diff
@@ -0,0 +1,318 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ffi';
+
+import 'package:ffi/ffi.dart';
+import 'package:tizen_interop/4.0/tizen.dart';
+import 'package:tizen_log/tizen_log.dart';
+
+/// The class for the property type of the bundle.
+class BundleTypeProperty {
```

Comment:
How about changing to internal? 
```suggestion
class _BundleTypeProperty {
```

Reply:
Done. Thanks.